packages/react-dom/src/client/ReactDOMRootFB.js

```diff
@@ -330,6 +339,38 @@ function legacyRenderSubtreeIntoContainer(
   return getPublicRootInstance(root);
 }
 
+export function findDOMNode(
```

Comment:
Should this go somewhere else since it'll still be on secret internals for now?

Reply:
I added essentially a re-export of findHostInstance in the internals instead which doesn't have the extra warnings and type checks. If you're using the internals you better know what you're doing.